src/cli/commands/team.js

```diff
@@ -38,14 +43,19 @@ function explodeScopeTeam(arg: string, requireTeam: boolean, reporter: Reporter)
   };
 }
 
-function warnDeprecation(reporter: Reporter) {
-  reporter.warn(`\`yarn team rm\` is deprecated. Please use \`yarn team remove\`.`);
+function warnDeprecation(reporter: Reporter, deprecationWarning: DeprecationWarning) {
+  reporter.warn(`\`yarn team ${deprecationWarning.deprecatedCommand}\` is deprecated.
```

Comment:
You should be using `reporter.lang` and add a language key for these so they can be translated.